src/rt.rs

```diff
@@ -11,6 +11,10 @@ use std::cell::Cell;
 use std::ptr;
 use std::sync::atomic::{AtomicBool, Ordering, ATOMIC_BOOL_INIT};
 
+extern "C" {
```

Comment:
```suggestion
#[cfg(target_os = "macos")]
extern "C" {
```

Reply:
I don't think this will compile as is - because `cfg!` cannot determine at compile time. However, I might be able to use a compile time inline `#[cfg]`. If we need an if/else chain, we can use something like https://github.com/alexcrichton/cfg-if - but I think in this case we don't need that and it'll be simpler to put some curly braces around the block! Updating momentarily!